libsnark/zk_proof_systems/plonk/tests/test_plonk.cpp

```diff
@@ -1095,6 +1095,18 @@ template<typename ppT> void test_plonk_gates_matrix_transpose()
     ASSERT_EQ(gates_matrix_transpose, example.gates_matrix_transpose);
 }
 
+template<typename ppT> void test_plonk_prepare_gates_matrix()
+{
+    using Field = libff::Fr<ppT>;
+    const size_t num_public_inputs = 8;
+    const std::vector<std::vector<Field>> gates_matrix_init =
+        plonk_prepare_gates_matrix<ppT>(num_public_inputs);
+    const std::vector<Field> PI_selector_vector{1, 0, 0, 0, 0};
```

Comment:
Assert on the size of `gates_matrix_init`?

Reply:
Fixed.